### PR TITLE
SDK-90: scheduler creation issue due to frequency format

### PR DESCRIPTION
**Pull request by abhisheksr8**

### CODE REVIEW

**Comment by mindprince** [comment]
Fix whitespace issues.

**Comment by mindprince** [comment]
Is there any backwards incompatibility impact here?

**Comment by abhisheksr8** [comment]
in SchedulerApi.java apart from create all other functions return InvokableBuilder type.
here i changed create also to return the same.
so we donot need CreateScheduleCommandBuilder to extend InvokableBuilder anymore, as none of the functions there need this.

**Comment by abhisheksr8** [comment]
regarding spaces, can you please point out areas where to fix as i didnt find any discrepancy there.

File: src/main/java/com/qubole/qds/sdk/java/details/CreateScheduleCommandBuilderImpl.java [hunk 1]
```diff
@@ -63,13 +61,27 @@ public CreateScheduleCommandBuilder end_time(String end_time)
         node.put("end_time", end_time);
         return this;
     }
-
+
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/CreateScheduleCommandBuilderImpl.java]: Trailing space

File: src/main/java/com/qubole/qds/sdk/java/details/CreateScheduleCommandBuilderImpl.java [hunk 2]
```diff
@@ -105,16 +117,17 @@ public CreateScheduleCommandBuilder dependency_info(DependencyInfo dependency_in
         node.putPOJO("dependency_info", dependency_info);
         return this;
     }
-
+
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/CreateScheduleCommandBuilderImpl.java]: Trailing space.

File: src/main/java/com/qubole/qds/sdk/java/details/InvocationCallbackBase.java [hunk 3]
```diff
@@ -52,7 +52,7 @@ public final RawInvokableBuilder raw()
     public final Future<T> invoke()
     {
         InvokeArguments<T> invokeArguments = getInvokeArguments();
-
+
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/InvocationCallbackBase.java]: Trailing space. This file shouldn't have any diff at all.

File: src/main/java/com/qubole/qds/sdk/java/details/SchedulerApiImpl.java [hunk 4]
```diff
@@ -84,21 +84,35 @@
     {
         return new GenericInvokableBuilderImpl<String>(client, RequestDetails.retry(), String.class, "scheduler", Integer.toString(scheduleId), "instances", Integer.toString(instanceId), "results");
     }
-
+
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/SchedulerApiImpl.java]: Trailing space.

File: src/main/java/com/qubole/qds/sdk/java/details/SchedulerApiImpl.java [hunk 5]
```diff
@@ -84,21 +84,35 @@
...
+    
+    @Override
+    public InvokableBuilder<Schedule> create(CreateScheduleCommandBuilder configBuilder)
     {
-        this.client = client;
+        RequestDetails entity = new RequestDetails(configBuilder.toString(), RequestDetails.Method.POST);
+        return new GenericInvokableBuilderImpl<Schedule>(client, entity, Schedule.class, "scheduler");
     }
-
+
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/details/SchedulerApiImpl.java]: trailing space.

**Comment by mindprince** [comment]
Pointed out some examples of trailing whitespaces. There are a lot of others as well.

**Comment by mindprince** [comment]
I pushed a new commit to master. If you pull that commit, it will fail the build when there are any whitespace issues.

**Pull request closed without merging**